articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md

```diff
@@ -0,0 +1,485 @@
+
+   - **Virtual network gateway** - Select the virtual network gateway you created earlier.
+
+   - **Local network gateway**- Select the local network gateway you created earlier.
+
+   - **Shared Key** - A mixture of letters and numbers, used to establish encryption for the connection. You must use the same shared key in both the virtual network and local network gateways. If your gateway device doesn't provide one, you can make one up here and provide it to your device.
+
+   - **Subscription** - This is your UKCloud for Microsoft Azure subscription.
+
+   - **Resource Group** - The resource group cannot be changed if you are adding a connection for an existing peer.
+
+   - **Location** - This will be `frn00006`, which is the location of the Azure Stack.
+
+     ![Add new connection](images/azs-browser-add-connection.png)
+
+6. Click **OK**.
+
+## Create a virtual network in Public Azure
+
+You'll also need to create a virtual network in Public Azure. This virtual network will be able to send and receive traffic through the virtual network gateway. You cannot associate a virtual network with more than one gateway.
+
+1. Log in to the [public Azure portal](https://portal.azure.com).
+
+2. In the favourites panel, select **Create a resource**.
+
+   ![Create a resource option in favourites panel](images/azs-public-browser-create-resource-button.png)
+
+3. In the **New** blade, select **Networking**.
+
+   ![Networking option in New blade](images/azs-public-browser-new-networking-button.png)
+
+4. In the **Featured** section, select **Virtual network**.
+
+   ![Create new networking resource](images/azs-public-browser-new-virtual-network.png)
+
+5. In the **Create virtual network** blade, enter the following information:
+
+   - **Name** - The name of the virtual network.
+
+   - **Address Space** - The virtual network's address range in CIDR notation.
+
+   - **Subscription** - This is your Microsoft Azure subscription.
+
+   - **Resource Group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
+
+   - **Location** - Select the location of the Azure resource.
+
+   - **Subnet Name** - The name of the first subnet within the virtual network.
+
+   - **Address Range** - The subnet's address range in CIDR notation (for example, 192.168.1.0). It must be within the address range of the virtual network. It should be noted that the address range of a subnet currently in use cannot be edited.
+
+   - **DDoS protection** - Select the DDoS protection you would like, for more information please see the [Azure DDoS Protection Standard overview](https://docs.microsoft.com/en-gb/azure/virtual-network/ddos-protection-overview).
+
+   - **Service endpoints** - This allows you to enable one or more service endpoints for more information please see [Virtual Network Service Endpoints](https://docs.microsoft.com/en-gb/azure/virtual-network/virtual-network-service-endpoints-overview).
```

Comment:
This allows you to enable one or more service endpoints. For more information, see [Virtual Network Service Endpoints](https://docs.microsoft.com/en-gb/azure/virtual-network/virtual-network-service-endpoints-overview).